doc/melange.rst

```diff
@@ -0,0 +1,296 @@
+.. _melange_main:
```

Comment:
```suggestion
:orphan:

.. _melange_main:
```

This will ensure that sphinx knows that this document is not part of any toctree.

Reply:
Update in latest version of the docs.